### PR TITLE
Add "more-info.md" story to "child-wave.md" story.

### PR DESCRIPTION
**Pull request by avasilescu**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/light-fire/run-away/shout/more-info/more-info.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+As you try to catch your breath from your marshmallow room escape, the boy sizes you up.
+
+"I'm trying to find this dog that I kind of lost. Well he ran away. Because I left the gate open..." he says.
+
+"You are in the forest near my town. What happened to you mister?"   
+
+Do you:
+
+[Help him find his dog] (find-dog/find-dog.md)
```
Comment (casey-collab) [inline english/light-fire/run-away/shout/more-info/more-info.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/light-fire/run-away/shout/more-info/more-info.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+[Explain to him about your escape] (explain-escape/explain-escape.md)
```
Comment (casey-collab) [inline english/light-fire/run-away/shout/more-info/more-info.md]: This link is also invalid.

File: english/light-fire/run-away/shout/more-info/more-info.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+[Ask him to take you to the town] (goto-town/goto-town.md)
```
Comment (casey-collab) [inline english/light-fire/run-away/shout/more-info/more-info.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by avasilescu** [comment]
Well, I seem to have screwed this all up now. I was trying to correct author email address in the last 3 commits. I ended up rebasing the entire branch. I'm just going to close this and redo it so it is cleaner.

**Pull request closed without merging**